requirements-test.txt

```diff
@@ -1,4 +1,5 @@
 # Test dependencies to avoid doctest import errors
 add-trailing-comma==2.4.0
 pyupgrade==3.3.1
+refurb==1.10.0;python_version>="3.10"
```

Comment:
I wonder how the tests are successful with python 3.9. Do you know?

Reply:
Sorry but i dont see any tests except doctests. wdym?

Reply:
Ah i see - refurb was not directly imported in the module unlike pu upgrade for example. So when pytest reads the module for doctest it doesn’t encounter an import error